zenodo/modules/records/data/objecttypes.json

```
@@ -712,4 +713,24 @@
     "type": "publication"
   },
   "csl": "article"
+},
+{
+  "$schema": "https://zenodo.org/schemas/records/objecttype-v1.0.0.json",
+  "internal_id": "publication-peerreview",
+  "id": "https://zenodo.org/objecttypes/publication/peerreview#",
+  "title": {
+    "en": "Peer review"
+  },
+  "title_plural": {
+    "en": "Peer reviews"
+  },
+  "schema.org": "https://schema.org/ScholarlyArticle",
+  "datacite": {"general": "PeerReview"},
+  "eurepo": "info:eu-repo/semantics/review",
+  "parent": {"$ref": "https://zenodo.org/objecttypes/publication"},
+  "openaire": {
+    "resourceType": "0015",
+    "type": "publication"
+  },
+  "csl": "article"
```

```suggestion
  "csl": "review"
```

Apparently this exsits, but I would leave it for a later deployment on Sandbox first to make sure that the citation rendering works.